test/execution-test-data/basicflowyamltest/basic_flow.flow

```diff
@@ -0,0 +1,40 @@
+# All flow level properties here
+config:
+  flow-level-parameter: value
+
+# This section defines the list of jobs
+# A node can be a job or a flow
+# In this example, all nodes are jobs
+nodes:
+  # Job definition
+  # The job definition is like a YAMLified version of properties file
+  # with one major difference. All custom properties are now clubbed together
+  # in a config section in the definition.
+  # The first line describes the name of the job
+  - name: shell_end
+    # Describe the type of the job
+    type: noop
+
+    # List the dependencies of the job
+    dependsOn:
```

Comment:
Per our earlier discussion, do you plan to change it to "dependedBy" later?

Reply:
Yes, I'll change it to "dependedBy" in a separate PR later.